src/pymor/models/iosys.py

```diff
@@ -63,8 +63,8 @@ class LTIModel(Model):
         The |Operator| D or `None` (then D is assumed to be zero).
     E
         The |Operator| E or `None` (then E is assumed to be identity).
-    cont_time
-        `True` if the system is continuous-time, otherwise `False`.
+    dt
+        `0` if the system is continuous-time, otherwise a positive number.
```

Comment:
I feel like we should mention here and also in the documentation of other classes that `dt` represents the sampling time if greater than `0`. E.g., "...,otherwise a positive number for the sampling time."

Reply:
True. Might be also good to mention that it is measured in seconds.